plugins/modules/type_info.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""This module allows users to retrieve information about a Linode Type."""
+
+from __future__ import absolute_import, division, print_function
+
+from typing import Any, Dict
+
+import ansible_collections.linode.cloud.plugins.module_utils.doc_fragments.type_info as docs
+from ansible_collections.linode.cloud.plugins.module_utils.linode_common_info import (
+    InfoModule,
+    InfoModuleAttr,
+    InfoModuleResult,
+)
+from ansible_specdoc.objects import FieldType
+from linode_api4 import LinodeClient, Type
+
+
+def _get_by_id(client: LinodeClient, params: Dict[str, Any]):
+    """
+    This function is intended to be passed into the ID get attribute.
+
+    NOTE: This is not implemented as a lambda because Type currently does not work with
```

Comment:
I've created an internal ticket to address this in the Python SDK but we can probably keep this workaround for now